packages/logs/lib/models/helpers.ts

```diff
@@ -57,10 +58,11 @@ export function getFormattedMessage(
         response: data.response || null,
         meta: meta || data.meta || null,
 
-        createdAt: data.createdAt || new Date().toISOString(),
-        updatedAt: data.updatedAt || new Date().toISOString(),
+        createdAt: data.createdAt || now.toISOString(),
+        updatedAt: data.updatedAt || now.toISOString(),
         startedAt: data.startedAt || null,
-        endedAt: data.endedAt || null
+        endedAt: data.endedAt || null,
+        expiresAt: data.operation ? data.expiresAt || new Date(now.getTime() + 7 * 86400 * 1000).toISOString() : null
```

Comment:
why 7 days by default. Can any operation last longer than 24hours?

Reply:
technically yes I believe, syncs can run as long as they need if the heartbeat works no?

Reply:
> syncs can run as long as they need if the heartbeat works no?

A sync can't run longer than 24 hours https://github.com/NangoHQ/nango/blob/master/packages/jobs/lib/workflows.ts#L6